src/store/actions.ts

```diff
@@ -1,6 +1,14 @@
+import { IActions } from "../interfaces/actions";
 import { ICategoryItem } from "../interfaces/category-Item";
-import { ADD_CATEGORY, LOAD_CATEGORIES, REQUEST_CATEGORIES } from "./types";
+import { ADD_CATEGORY, LOAD_CATEGORIES, REQUEST_ADD_CATEGORIES, REQUEST_CATEGORIES } from "./types";
```

Comment:
please, use singlequotes everywhere   except  it is pure HTML

Reply:
+